content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/150-introspection.mdx

```diff
@@ -338,6 +679,142 @@ const user = await prisma.myUser.create({
 
 Learn more about this on the [Configuring your Prisma Client API](/concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names) page.
 
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'planetscale']}>
+
+As a next step, you will introspect your database. The result of the introspection will be a [data model](/concepts/components/prisma-schema/data-model) inside your Prisma schema.
+
+Run the following command to introspect your database:
+
+```terminal copy
+npx prisma db pull
+```
+
+This commands reads the `DATABASE_URL` environment variable that's defined in `.env` and connects to your database. Once the connection is established, it introspects the database (i.e. it _reads the database schema_). It then translates the database schema from SQL into a Prisma data model.
+
+After the introspection is complete, your Prisma schema file was updated:
+
+![Introspect your database with Prisma](https://res.cloudinary.com/prismaio/image/upload/v1628761155/docs/f7itiYw.png)
+
+The data model now looks similar to this (note that the fields on the models have been reordered for better readability):
```

Comment:
Why do we not just order the columns in the tables the way we want them to appear here?

Reply:
Actually, looking at this again the ordering of `Post`, `Profile`, `User` is the same in the SQL and the Prisma schema, right? So I can just delete this note?

(This was a copypaste from the instructions for a different database.)

Reply:
Possible. Not that the note says `fields on the models` though - so this might refer to the order of the individual fields inside the 3 models. Best way to confirm is to actually run through the steps you describe and make sure the result is exactly what we then use as the example. (Which should of course also be the goal, or users will be confused)

Reply:
I've run through the steps again (with the new @@index annotations) and checked that the fields and models are in the correct order. So I've deleted the note.